code/js/controllers/PlexController.js

```diff
@@ -5,11 +5,11 @@
 
   var controller = new MouseEventController({
     siteName: "Plex.tv",
-    play: "i[class^=plex-icon-player-play-]",
-    pause: "i[class^=plex-icon-player-pause-]",
-    playNext: "i[class^=plex-icon-player-next-]",
-    playPrev: "i[class^=plex-icon-player-prev-]",
-    mute: "i[class^=plex-icon-player-volume-]",
+    play: "button[aria-label='Play']",
```

Comment:
When you're on a playlist page this grabs the play button in the top toolbar instead of the player button.  Maybe the following would work?

```javascript
    play:"button[data-qa-id='resumeButton']",
    pause:"button[data-qa-id='pauseButton']",
    playNext:"button[data-qa-id='nextButton']",
    playPrev:"button[data-qa-id='previousButton']",
    mute:"button[data-qa-id='volumeButton']"
```

Reply:
Oh, good catch! 

I'd rather not use attributes that appear to be intended for QA purposes (and will likely get stripped at some later point). But we can scope it to the buttons with those labels that are just in the player bar. Let me see if I can find an appropriate selector...